src/style.css

```diff
@@ -0,0 +1,65 @@
+html {
+  font-family: "Roboto", "Arial", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
```

Comment:
try to use justify-content: space-between;;

Reply:
space-beetween breaks my code

Reply:
flex-end looks weird